kedro_vertexai/utils.py

```diff
@@ -83,3 +85,31 @@ def _generate_and_save_dynamic_config(provider: DynamicConfigProvider, context_h
     provider.merge_with_existing(existing_config, dynamic_config)
     logger.info(f"Saving dynamic config {target_path} [{type(provider).__name__}]")
     save_yaml(dynamic_config, target_path)
+
+
+def docker_build(path: str, image: str):
+    rv = subprocess.run(
+        [
+            "docker",
+            "build",
+            path,
+            "-t",
+            image,
+        ]
+    ).returncode
+    if rv:
+        logger.error("Docker build has failed.")
```

Comment:
Will we see the output from the command here? (stdout/stderr)
If not, it's worthy to add it, otherwise user will not know what to fix to make it run well.

(same for push)

Reply:
Error is displayed fine, the output I see during the testing:
[console_out.txt](https://github.com/getindata/kedro-vertexai/files/10030890/console_out.txt)